extensions/libraries/redcore/api/payment/document/document.php

```diff
@@ -97,19 +97,38 @@ public function render($cache = false, $params = array())
 		// Get the payment string from the buffer.
 		$content = $this->getBuffer();
 
-		$app->setHeader('Status', $this->payment->statusCode . ' ' . $this->payment->statusText, true);
-		$app->setHeader('Server', '', true);
-		$app->setHeader('X-Runtime', $runtime, true);
-		$app->setHeader('Access-Control-Allow-Origin', '*', true);
-		$app->setHeader('Pragma', 'public', true);
-		$app->setHeader('Expires', '0', true);
-		$app->setHeader('Cache-Control', 'must-revalidate, post-check=0, pre-check=0', true);
-		$app->setHeader('Cache-Control', 'private', false);
-		$app->setHeader('Content-type', $this->_mime . '; charset=UTF-8', true);
-		$app->setHeader('Content-length', strlen($content), true);
-		$app->setHeader('Content-Language', $language, true);
-
-		$app->sendHeaders();
+		if (version_compare(JVERSION, '3.1') >= 0)
```

Comment:
I would rather have a separate function in RApi that can add header depending on the version so you just call the same function instead of separating it here, because we have that for SOAP and Webservices too. Can you do it like that please?